src/RefractAPI.cc

```diff
@@ -339,82 +379,53 @@ std::unique_ptr<IElement> PayloadToRefract( //
     if (!payload.node->description.empty())
         content.push_back(CopyToRefract(MAKE_NODE_INFO(payload, description)));
 
-    std::unique_ptr<IElement> payloadAttributeElement = payload.node->attributes.empty() ? //
-        nullptr :
+    auto unexpandedAttrs = payload.node->attributes.empty() ? //
```

Comment:
I'd consider calling this something like `unexpandedDataStructures` because `attributes` is a bit of an overloaded term (API Blueprint syntax "attributes", MSON AST "attributes", API Elements "attributes") which confused me a for alittle bit.